src/app/actions.ts

```diff
@@ -0,0 +1,37 @@
+"use server";
+
+export async function createEvent(data) {
+  const res = await fetch("http://localhost:3333/event", {
```

Comment:
for the localhost port, can you add that as a .env variable? then we can switch out easily on the real backend url when deployed

Reply:
added

Reply:
done